Makefile

```diff
@@ -64,12 +64,16 @@ fmt:
 
 # Do all code/CRD generation at once, with timestamp file to check out-of-date.
 GEN_TIMESTAMP=.zz_generate_timestamp
+MANIFESTS=manifests/$(OCP_VERSION)
```

Comment:
Why do we have to hardcode the version if we can figure it out by inspecting the manifests dir?  This inhibits our desire to make it easier to embrace feature branches with openshift build CI

Reply:
OK, I'll revert that for this PR. I have some questions/thoughts but will raise them separately.